data/mods/Tanks/items.json

```diff
@@ -351,7 +351,7 @@
   "recoil": 15,
   "durability": 8,
   "burst": 5,
-  "clip_size": 5,
+  "clip_size": 20,
```

Comment:
Ah, thank you. That will be even harder for players to tolerate loading one at a time, but oh well.